CMakeLists.txt

```diff
@@ -232,6 +232,10 @@ target_include_directories(
 	${CMAKE_CURRENT_SOURCE_DIR}/src/generated
 )
 
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+	target_compile_definitions(lcf PUBLIC _DEBUG=1)
```

Comment:
Can we call this something like `LCF_DEBUG`?

Reply:
okay.
I'm currently thinking about making the whole skip debug printout always on even in Release as this indicated a problem in our parser (unknown chunk).